inst/COPYRIGHTS

```diff
@@ -11,12 +11,11 @@ The package XLConnect includes other open source software components. The follow
 
    Related JARs: poi-*.jar, poi-ooxml-*.jar
 
-
 Transitive dependencies include:
```

Comment:
Not sure if we need/should extend the list of transitive dependencies. I guess we could do one line for all Apache projects such as POI, Commons Compress, Commons IO, XMLBeans, etc.

Reply:
I will also remove the "related JARs" as well for the dependencies that are downloaded on-the-fly.